xbmc/guilib/GUIBaseContainer.cpp

```diff
@@ -404,14 +404,14 @@ bool CGUIBaseContainer::OnAction(const CAction &action)
   case ACTION_SHOW_INFO:
     if (m_listProvider)
     {
-      int selected = GetSelectedItem();
+      const int selected = GetSelectedItem();
       if (selected >= 0 && selected < static_cast<int>(m_items.size()))
       {
-        m_listProvider->OnInfo(m_items[selected]);
-        return true;
+          if (m_listProvider->OnInfo(m_items[selected]))
+            return true;
```

Comment:
Indentation looks wrong.

Reply:
well it was done by `clang-format` :shrug:

Reply:
> well it was done by clang-format 🤷

I had such cases as well, were clang-format obviously did it wrong. I guess it happens if the file contains many style errors then clang-format gets "confused". In your case the if now is indented 3 spaces, which is obviously wrong.